tests/test_import.py

```diff
@@ -22,3 +23,8 @@ def test(self):
     def test_unsupported(self):
         ret = main(['import', 'unsupported://path', 'import_unsupported'])
         self.assertNotEqual(ret, 0)
+
+class TestDefaultOutput(TestDvc):
+    def test(self):
+        ret = main(['import', self.FOO])
```

Comment:
This is a circular dependency though :) It is an equivalent of `dvc run -d foo -o foo cp foo foo`. This will be tested in the DAG itself with https://github.com/iterative/dvc/issues/1285. Let's rather create a temporary file(see self.mkdtem() or tempfile.NamedTemporaryFile()) with some content and then import it by the abs path and then check that the file name is the one we are expecting.